emily/handler/src/api/handlers/internal.rs

```diff
@@ -62,29 +62,31 @@ async fn set_api_state_status(
                 if new_reorg_tip.key == current_reorg_tip.key {
                     return Ok(None);
                 } else {
-                    let err_msg: String = format!(
-                        "Trying to reorg with new chaintip {new_reorg_tip:?} while the api is reorganizing around the chaintip {current_reorg_tip:?}"
-                    );
-                    warn!(err_msg);
-                    return Err(Error::InconsistentState(Inconsistency::ItemUpdate(err_msg)));
+                    let message =
+                        "Trying to reorg with new chaintip while the API is already reorganizing";
+                    warn!(?new_reorg_tip, ?current_reorg_tip, message);
+                    return Err(Error::InconsistentState(Inconsistency::ItemUpdate(message)));
                 }
             }
         };
 
         debug!(
-            "Changing Api state from [{original_api_state:?}] to [{api_state:?}]. Attempt {attempt_number} of maximum {MAX_SET_API_STATE_ATTEMPTS_DURING_REORG}."
+            ?api_state,
+            ?original_api_state,
+            attampt = %attempt_number,
```

Comment:
```suggestion
            attempt = %attempt_number,
```

Reply:
Fixed in https://github.com/stacks-sbtc/sbtc/pull/1627/commits/701270aa69c471be019d1c9da67a5ac88cd7d8e5